helpers/helper-translation-discussion.php

```diff
@@ -214,6 +214,23 @@ public function sanitize_comment_topic( $comment_topic ) {
 		return $comment_topic;
 
 	}
+
+	public function sanitize_comment_locale( $comment_locale ) {
+		$gp_locales     = new GP_Locales();
+		$all_gp_locales = array_keys( $gp_locales->locales );
+
+		if ( ! in_array( $comment_locale, $all_gp_locales ) ) {
+			$comment_locale = 'unknown';
+		}
+		return $comment_locale;
+	}
+
+	public function sanitize_translation_id( $translation_id ) {
+		if ( ! is_numeric( $translation_id ) ) {
+			wp_die( 'Invalid translation ID' );
```

Comment:
I think it's ok to `wp_die` here but we could make an effort to actually check if the translation id exists, no? `is_numeric( -10 )` is also `true` which can't be right. I think it's something like `GP::$translation->get( $translation_id )`

Reply:
This makes sense, thanks! Even though an invalid transaction ID in the url will throw a 404. A user can load a valid a page with valid transaction ID in the url and do a POST with invalid transaction ID.